annotations/forms.py

```diff
@@ -20,6 +20,26 @@ def __init__(self, *args, **kwargs):
 		self.helper = FormHelper(self)
 
 
+class RegistrationForm(forms.Form):
+
```

Comment:
@shivamprakash Were you able to do this? Shouldn't take more than a minute or two